src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -840,8 +840,8 @@ cli_enable_ssl(int argc, char **argv)
 			}
 			else
 			{
-				log_info("  pg_autoctl service is not running, changes "
-						 "will only apply at next start of pg_autoctl");
+				log_error("  pg_autoctl service is not running, changes "
```

Comment:
Should this one be a WARN message really? I mean there's a use case to changing the SSL settings while in maintenance or something...